### ISSUE TITLE
Returning to page not visible

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#200]
Screengrab/videos (insert file names here)
2020 11 07, Returning to page not visible.mp4

Link
https://www.boston.gov/my-neighborhood

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.7 > Safari

Trusted Tester Test

Brief Description
Returning to page not visible

Steps to reproduce
1)	00:00, Open the link in a private browser window, with a search and search results visible.
2)	00:05, Use the tab key to navigate down the list.
3)	00:08, Select an item and show the neighborhood details
4)	00:13, Click on one of the options, such as City Services
5)	00:15, Tab to navigate to the “Back to Results” button, and activate the button.
6)	00:16, BUG:  The focus does not return to the results of the search and additional navigation is needed

Desired Result
•	Focus is visible when navigating between screens.

Actual Result
•	Focus is not visible when activating “Back to Results”
[2020 11 07, Returning to page not visible.zip](https://github.com/CityOfBoston/Iterators/files/5514485/2020.11.07.Returning.to.page.not.visible.zip)

**Comment by reillyzlab** [comment]
Not sure if this is still an issues. @david-iterators can you verify and/or provide more clarity on what the issue is.
